pkg/tsdb/postgres/postgres_test.go

```diff
@@ -190,6 +189,7 @@ func TestPostgres(t *testing.T) {
 		ConnectionString:  "",
 		DSInfo:            dsInfo,
 		MetricColumnTypes: []string{"UNKNOWN", "TEXT", "VARCHAR", "CHAR"},
+		RowLimit:          1000000,
```

Comment:
i see that you are changing the hard coded value to the setting, i don't recall anymore whether we have test regarding it, probably we could by making a rowlimit small number as 1 here and check the response length?

Is this PR related to the EPIC of limit the datasource response size?

Reply:
> i see that you are changing the hard coded value to the setting, i don't recall anymore whether we have test regarding it, probably we could by making a rowlimit small number as 1 here and check the response length?

Yeah I was thinking about doing that and since you confirmed it I'll go ahead and add such tests.

> Is this PR related to the EPIC of limit the datasource response size?

Yes